examples/user-config/maven/README.md

```diff
@@ -0,0 +1,17 @@
+# Camel K Maven configuration examples
```

Comment:
Good. Let's move it under /examples/maven. user-config directory was thought for `kamel run` parameters.